### ISSUE TITLE
Groups are not named correctly and actual shortcuts of apps doesn't show any name

### ISSUE THREAD

**Issue by alfonso73** [tjackenpacken/taskbar-groups#196]
Hi all, despite the fact that i've named correctly everything the groups are named just "Group" and no name is shown on actuall apps shortcuts. BTW would be great to have te feature to give apps shortcuts custom names. thanks

**Comment by kamrul02** [comment]
I do not think the software hass this feature yet - showing names under each icon of the application itself. However, I agree that it would be extremely helpful.

**Comment by Harm10** [comment]
I would also like some naming to be visible. A regular app on the task bar has a mouse over name so something like that would be useful. (Group names are working properly on my side).

**Comment by nima-amin** [comment]
Yes, dito, I have created a group in which I put several Windows system parts and tools which to find when needed takes too much time (task manager, windows explorer, system information, Systemsteuerung, Gerätemanager (sorry, I don't know the English names of the latter two, but their icons look so similar, it would be helpful to have tooltips when hovering over them also in TaskbarGroups.

**Comment by gaveitatry2** [comment]
I downloaded TaskbarGroups v0.2.0.0 today on Feb 10, 2023 for Windows 10. I wanted to use this program, but unfortunately I could not. I tested the program out by adding 11 shortcuts. Each shortcut was to a Ren'Py Visual Novel game. And every shortcut had the same icon. When I pinned the taskbar group to the taskbar, I was sad to see that I could not distinguish any of the 11 shortcuts from each other because there was no tool tip to tell me the shortcut name. The only workaround I can see for this is to create custom icons for each shortcut but that's too much work and headache. Hopefully a future version of this program will have shortcut program names on tooltips when you hover over the shortcuts.
